deps/wazuh_testing/wazuh_testing/qa_ctl/configuration/config_generator.py

```diff
@@ -131,9 +131,9 @@ class QACTLConfigGenerator:
         }
     }
 
-    def __init__(self, tests=None, wazuh_version=None, qa_branch='master',
+    def __init__(self, modules_data=None, wazuh_version=None, qa_branch='master',
                  qa_files_path=join(gettempdir(), 'wazuh_qa_ctl', 'wazuh-qa'), systems=None):
-        self.tests = tests
+        self.modules_data = modules_data
```

Comment:
Rename all `modules_data` names to `test_modules_data` to make reference to the `test` word.

A "module" can be confusing, specially in python

Reply:
Solved https://github.com/wazuh/wazuh-qa/pull/2640/commits/5c8ec5b57dea53781df54dfee43a63ab27e113ed